kythe/typescript/testdata/function.ts

```diff
@@ -32,6 +32,21 @@ function test(a: number, num: Num): Num {
 //- @test ref F
 test(3, {num: 3});
 
+
+// Test comoputed function names
+//- @#0"fn" defines/binding OFnStr
+const fn = 'fn';
+let o = {
+  //- @"['ofn']" defines/binding OFn
```

Comment:
I can't see how this anchor could possibly match—there is no corresponding text.

Reply:
Doh, that's on me. Thanks!